src/components/LoginForm/index.js

```diff
@@ -24,13 +25,8 @@ function LoginForm({email, password, handleSubmit, handleChange}){
             >
                 Sign In
             </button>
-             <button 
-                type="button"
-             >
-                Create User
-            </button>
+            <Link to={`/welcome/${email}`}>Create user </Link>
```

Comment:
poner el email del usuario en la url de esta forma no es muy recomendado. lo pasaría mejor o por query string o lo pasaría por un state del link usando https://reactrouter.com/web/api/Link/to-object